great-mystery-great-whore/original/080-chapter-79.adoc

```diff
@@ -52,7 +52,8 @@ and that which came out of him, is of him, and rejoiceth in him.
 Why wilt thou lie, in saying the Quakers deny the word?
 We say, it sanctifies them.
 
-His lies.
+[.discourse-part]
+_His lies._
```

Comment:
I wasn't sure if I should do all this here, but felt like it needed something.

Reply:
Works for me 👍